### PR TITLE
[SEMI-MODULAR] LOOC mutes now exist.

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

LOOC can now be muted by an admin for those who have decided to abuse it.

## Why It's Good For The Game

This was requested by an admin.

## Changelog
:cl: ZephyrTFA
admin: LOOC can now be muted. How you managed to earn a LOOC mute is beyond me though.
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Can you make this a separate ban type? You know how we have OOC bans, make an LOOC ban? I already set up a system for adding new bans!

**Comment by ZephyrTFA** [comment]
Yea; can do, give me a bit.

**Comment by SpaceVampire** [comment]
Linking OOC and LOOC bans is an bad idea.
Given that LOOC is important when it comes to like, prefs and stuff of that nature.
If someone is causing trouble with it, then they should just get a server-ban honestly.
OOC bans are mostly just given for like, icky ocky, or salting using it.
I really can't think a reason you would LOOC ban someone.

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -106,6 +106,9 @@
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_PRAY]'><font color='[(muted & MUTE_PRAY)?"red":"blue"]'>PRAY</font></a> | "
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_ADMINHELP]'><font color='[(muted & MUTE_ADMINHELP)?"red":"blue"]'>ADMINHELP</font></a> | "
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_DEADCHAT]'><font color='[(muted & MUTE_DEADCHAT)?"red":"blue"]'>DEADCHAT</font></a>\]"
+		// Skyrat Edit Addition - Mute LOOC
+		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_LOOC]'><font color='[(muted & MUTE_LOOC)?"red":"blue"]'>DEADCHAT</font></a>\]"
```
Comment (Gandalf2k15) [inline code/modules/admin/admin.dm]:
```suggestion
		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_LOOC]'><font color='[(muted & MUTE_LOOC)?"red":"blue"]'>LOOC</font></a>\]"
```

**Comment by Gandalf2k15** [comment]
Looks good.

**Pull request merged**